server/src/main/scala/za/co/absa/atum/server/api/service/PartitioningServiceImpl.scala

```diff
@@ -61,6 +58,46 @@ class PartitioningServiceImpl(partitioningRepository: PartitioningRepository)
       }
   }
 
+  override def getPartitioningCheckpoints(checkpointQueryDTO: CheckpointQueryDTO): IO[ServiceError, Seq[CheckpointDTO]] = {
+    repositoryCall(
+      partitioningRepository.getPartitioningCheckpoints(checkpointQueryDTO), "getPartitioningCheckpoint"
+    ).mapBoth(error => ServiceError(error.message), {
```

Comment:
is this map needed? `error => ServiceError(error.message)` - it's already in `repositoryCall`

Reply:
I guess not.